debian/control

```diff
@@ -916,6 +912,28 @@ Description: MySQL compatibility links to mariadb-server binaries/scripts.
  This package is also useful for users with strong  background in MySQL
  maintenance, but wants to switch to MariaDB
 
+Package: mariadb-server-galera
+Architecture: any
+Depends: galera-4 (>= 26.4),
+         gawk,
+         iproute2 [linux-any],
+         lsof [linux-any],
+         mariadb-server,
+         rsync,
+         socat,
+         ${misc:Depends},
+         ${perl:Depends},
+         ${shlibs:Depends}
+Conflicts: mariadb-server (<< 11.3)
+Breaks: mariadb-server (<< 1:11.3)
+Description: MariaDB database scripts needed by Galera cluster
+ MariaDB Galera Cluster is a virtually synchronous multi-primary
+ cluster for MariaDB. It is available on Linux only, and only supports
+ the InnoDB storage engine (although there is experimental support for
+ MyISAM and, from MariaDB 10.6, Aria.
```

Comment:
"from MariaDB 10.6" looks odd in a 11.3 package - omit.

Reply:
Ok i've reworded. Should read what copy-paste